### ISSUE TITLE
Accessibility does not work for Android devices for react native modals

### ISSUE THREAD

**Issue by peterjwlee24** [react-native-modal/react-native-modal#525]
I am using react native modals , and I noticed for android devices accessibility does not work well. For example, inside the modal, talk back does not correctly focus on the items in the modal. Has this problem been addressed or has someone found a work around this? Thank you!

**Comment by mmazzarolo** [comment]
Hi! 
I'm closing this issue because it's a question and not a bug report. Please don't take it personally, I'm just trying to make it easier to maintain this project by separating questions from bug reports 🙏.  
If you're still looking for feedback/answers, could you please ask your question in a new [discussion](https://github.com/react-native-modal/react-native-modal/discussions)?  
Thank you!
